code/__HELPERS/experimental.dm

```diff
@@ -68,6 +68,7 @@ var/list/exclude = list("inhand_states", "loc", "locs", "parent_type", "vars", "
  * Example call: returnToPool(src)
  */
 /proc/returnToPool(const/atom/movable/AM)
+	if(!AM) return // Nothing to delete/pool anyways, so no action needed
```

Comment:
Yeah needs to be an assert or atleast
~~~
if(!AM)
    CRASH("returnToPool got a null arg")
~~~